randovania/games/am2r/json_data/Distribution Center.txt

```diff
@@ -1742,55 +1708,31 @@ Extra - map_name: rm_a5b21
   * Layers: default
   * Normal Door to Facility Storage Tower Route North/Door to Facility Storage Tower West; Dock Lock Rando incompatible with: Spider Ball Door, Screw Attack Door
   * Extra - instance_id: 136414
-  > Door to Facility Storage Tower Route South
+  > Center
       Trivial
-  > Door to Distribution Center Exterior West
-      Any of the following:
-          Gravity Suit or Power Grip or Movement (Beginner) or Walljump (Beginner)
-          All of the following:
-              # IBJ
-              Infinite Bomb Jumping (Intermediate) and Can Use Bombs
-          All of the following:
-              # CBJ
-              Charged Bomb Jump (Beginner) and Can Use Charged Bomb Jump
 
 > Door to Facility Storage Tower Route South; Heals? False
   * Layers: default
   * Normal Door to Facility Storage Tower Route South/Door to Facility Storage Tower West; Dock Lock Rando incompatible with: Spider Ball Door, Spider Ball Door
   * Extra - instance_id: 136415
-  > Door to Facility Storage Tower Route North
-      Any of the following:
-          Power Grip or Walljump (Beginner)
-          Gravity Suit and Power Grip Wall
-          All of the following:
-              # IBJ
-              Infinite Bomb Jumping (Intermediate) and Can Use Bombs
-  > Door to Distribution Center Exterior West
-      Any of the following:
-          Power Grip or Walljump (Beginner)
-          Gravity Suit and Power Grip Wall
-          All of the following:
-              # IBJ
-              Infinite Bomb Jumping (Intermediate) and Can Use Bombs
-          All of the following:
-              # CBJ
-              Charged Bomb Jump (Beginner) and Can Use Charged Bomb Jump
+  > Center
+      Underwater Power Grip Wall
 
 > Door to Distribution Center Exterior West; Heals? False
   * Layers: default
   * Normal Door to Distribution Center Exterior West/Door to Facility Storage Tower West; Dock Lock Rando incompatible with: Spider Ball Door, Screw Attack Door
   * Extra - instance_id: 136413
+  > Center
+      Trivial
+
+> Center; Heals? False
+  * Layers: default
   > Door to Facility Storage Tower Route North
-      Any of the following:
-          Gravity Suit or Power Grip or Movement (Beginner) or Walljump (Beginner)
-          All of the following:
-              # IBJ
-              Infinite Bomb Jumping (Intermediate) and Can Use Bombs
-          All of the following:
-              # CBJ
-              Charged Bomb Jump (Beginner) and Can Use Charged Bomb Jump
+      Movement (Beginner) or Underwater Power Grip Wall
```

Comment:
While this is technically correct, i feel like it'd be clearer to have to doors directly connect to each other with movement. makes it clearer that the movement is the jumping from one door to another, not jumping up.